src/swap/SwapScreen.tsx

```diff
@@ -58,13 +62,35 @@ const DEFAULT_SWAP_AMOUNT: SwapAmount = {
 
 type Props = NativeStackScreenProps<StackParamList, Screens.SwapScreenWithBack>
 
+function getNetworkFee(quote: QuoteResult | null, networkId?: NetworkId) {
+  // TODO remove this block once we have viem enabled for everyone. this block
+  // only services the contractKit (Celo) flow. the fee will be displayed in
+  // fiat value and will be very low, since it's an approximation anyway we'll
+  // simplify this logic to just use the fees from the quote.
+  if (quote && !quote.preparedTransactions) {
```

Comment:
nit: code golf
```suggestion
  if (!quote?.preparedTransactions) {
```

Reply:
If `quote` is null we don't want to enter in the if block.
`!quote?.preparedTransactions` will return true if quote is null, right?

Reply:
yes, i need the quote here but no prepared transactions :)